code/modules/admin/admin_verbs.dm

```diff
@@ -1053,3 +1056,12 @@ GLOBAL_LIST_INIT(view_runtimes_verbs, list(
 			show_blurb(about_to_be_banned, 15, message, null, "center", "center", message_color, null, null, 1)
 			log_admin("[key_name(src)] sent an admin alert to [key_name(about_to_be_banned)] with custom message [message].")
 			message_admins("[key_name(src)] sent an admin alert to [key_name(about_to_be_banned)] with custom message [message].")
+
+
+/client/proc/export_current_character()
+	set name = "Export Character DMI/JSON"
+	set category = "Admin"
+
+	if ishuman(mob)
```

Comment:
wait, this compiles?
```suggestion
	if(ishuman(mob))
```

Reply:
Ah yeah. The `ishuman` macro is surrounded by parens. Fixed.